### PR TITLE
feat: Convert CLI Output to Tabulate

### PR DESCRIPTION
**Pull request by stratus-ss**
feat: This PR removes the manual formatting that was being done for the text tables. It now uses tabulate

As a result tests needed to be altered as did Analyzer so that the dataFrame is now being passed in to the print function instead of an adjusted string. In addition, this meant that tests needed to be changed to account for the new output. Further to this there were several functions that were no longer needed and therefore were removed as were the associated unused arguments

### CODE REVIEW

File: vminfo_parser/analyzer.py [hunk 1]
```diff
@@ -320,13 +311,12 @@ def sort_by_disk_space_range(
         for column_to_drop in drop_columns:
             sorted_range_counts_by_environment.drop(column_to_drop, axis=1, inplace=True)
 
-        return column_padding, sorted_range_counts_by_environment
+        return sorted_range_counts_by_environment
```
Comment (csfreak) [inline vminfo_parser/analyzer.py:314]: The annotations on the method signature, and the docstring need to be updated to match the new return

File: vminfo_parser/clioutput.py [hunk 2]
```diff
@@ -86,110 +87,35 @@ def format_series_output(self: t.Self, counts: pd.Series) -> None:
...
         This function displays a header and the formatted data, optionally filtered by the operating system.
 
         Args:
-            col_widths (dict): A dictionary containing the widths for each column in the output.
             formatted_df_str (str): A string representation of the formatted disk space data.
             os_filter (Optional[str]): An optional filter to display specific operating system information.
 
         Returns:
             None
         """
```
Comment (csfreak) [inline vminfo_parser/clioutput.py:105]: The docstring defines the data coming in as a string, the signature defines a DataFrame

**Review by csfreak — CHANGES_REQUESTED** [review]

File: vminfo_parser/analyzer.py (hunk 1 above)
Comment (stratus-ss) [reply vminfo_parser/analyzer.py:314]: fixed

File: vminfo_parser/clioutput.py (hunk 2 above)
Comment (stratus-ss) [reply vminfo_parser/clioutput.py]: fixed

**Review by csfreak — APPROVED** [review]

**Pull request merged**